src/brevitas/export/manager.py

```diff
@@ -254,8 +205,6 @@ def _cache_inp_out(cls, module, *args, **kwargs):
         module.apply(lambda m: _override_inp_caching_mode(m, enabled=True))
         module.apply(lambda m: _override_out_caching_mode(m, enabled=True))
         with ExitStack() as stack:
```

Comment:
If we don't add anything to the stack, we can remove the context manager and leave the forward pass as part of the function